math/FunMath.cpp

```diff
@@ -1,5 +1,17 @@
 #include "FunMath.h"
 
+using namespace std;
+
+
+
```

Comment:
You need a function comment describing what this function does, what parameters it uses and what its return value is.